packages/dht/test/end-to-end/WebSocketConnectionRequest.test.ts

```diff
@@ -9,7 +9,7 @@ describe('WebSocket IConnection Requests', () => {
     const epPeerDescriptor: PeerDescriptor = {
         kademliaId: PeerID.fromString('3').value,
         type: NodeType.NODEJS,
-        websocket: { ip: '127.0.0.1', port: 10021 }
+        websocket: { host: '127.0.0.1', port: 10021, tls: false }
```

Comment:
Add a test case (to some file) where we actually use tls connections?

Reply:
https://linear.app/streamr/issue/NET-1077/test-case-for-tls-connections-in-trackerless-network-client